### DISCUSSION TITLE
KSADS Coding of 888

### DISCUSSION THREAD

**Discussion by Jennifer0999** [now-i-know-my-abcd/docs#73] [Stats & Code Help]
Hi, I'm new to the KSADS data in ABCD. For the KSADS diagnoses fields, 888 = The Module is not administered, e.g. individual did not answer screening items affirmatively and value missing due to branching logic. is it appropriate to code the '888' responses as '0' or NO for the diagnosis? Thank you!

**Comment by lmurray89** [comment]
Here is my guess (not looking at ABCD specifically). Usually Ksads has screening questions, where if the subject doesnt endorse those items, the rest of the module isn't given. If that is the case with abcd, then you could feel pretty confident in coding as 0=NO. If it wasn't administered at all for some reason, then I don't think 0 is really appropriate because you truly don't know
